runelite-client/src/main/java/net/runelite/client/plugins/reportbutton/ReportButtonPlugin.java

```diff
@@ -71,6 +82,39 @@ ReportButtonConfig provideConfig(ConfigManager configManager)
 		return configManager.getConfig(ReportButtonConfig.class);
 	}
 
+	private int[] cacheVarb;
+
+	@Subscribe
+	public void onVarbitChange(VarbitChanged event)
+	{
+		if (client.getSetting(Varbits.IN_RAID) == 1)
+		{
+			raidReady = true;
+		}
+		else
+		{
+			raidReady = false;
+			raidStartTime = null;
+			raidEndTime = null;
+		}
+	}
+
+	@Subscribe
+	public void onGameMessage(ChatMessage event)
+	{
+		if (raidReady && event.getType() == ChatMessageType.CLANCHAT_INFO)
+		{
+			if (event.getMessage().contains(RAID_START))
+			{
+				raidStartTime = Instant.now();
```

Comment:
reset raidEndTime to null here as well

Reply:
Also maybe reset raidEndTime and raidStartTime on logout?

Reply:
if you dc from a raid, you can still reconnect.